### PR TITLE
removing Transaction field from the PayloadAttributes structure

### PR DESCRIPTION
**Pull request by AnkushinDaniil**
## Summary by CodeRabbit

- **Bug Fixes**
  - Improved error handling during payload validation to enhance reliability.

- **Refactor**
  - Updated methods for better handling of payload attributes and transaction adaptation.
  - Removed outdated fields in payload structure, streamlining transaction processing.

These changes result in a more reliable and efficient transaction validation experience for users.

### CODE REVIEW

**Comment by AnkushinDaniil** [comment]
@joshklop review request

**Review by joshklop — CHANGES_REQUESTED** [review]
Thinking about this again, I feel we can simplify things a bit more. Let's remove the `Transactions` field instead of the `CosmosTxs` field. Existing references to `Transactions` can be replaced with `CosmosTxs`.

If you have different thoughts, let me know. My quick implementation locally seems to work well.

File: engine/engine.go [hunk 1]
```diff
@@ -208,8 +208,6 @@ func (e *EngineAPI) ForkchoiceUpdatedV3(
 		ParentBeaconBlockRoot: pa.ParentBeaconBlockRoot,
 		ParentHash:            fcs.HeadBlockHash,
 		Height:                e.blockStore.HeadBlock().Header.Height + 1,
-		Transactions:          pa.Transactions,
-		CosmosTxs:             cosmosTxs,
```
Comment (joshklop) [inline engine/engine.go:212]: We should keep the `cosmosTxs` field.

Comment (AnkushinDaniil) [reply engine/engine.go:212]: The problem was fixed by re-running `make setup-e2e'.

**Review by joshklop — APPROVED** [review]

**Pull request merged**